project_testing/housing_characteristics/Lighting Use.tsv

```diff
@@ -0,0 +1,2 @@
+Option=95% Usage	Option=100% Usage
+1	0
```

Comment:
Since this is the _testing_ project, meant to exercise a range of different options, I would suggest giving equal probability to each option. You did this for some of the new TSVs in the testing project but not others.

Reply:
💯 Yup, I got distracted with the different projects. Fixed.